### DISCUSSION TITLE
Keep a Playlist Updated

### DISCUSSION THREAD

**Discussion by ArthurOllier** [NickvisionApps/Parabolic#676] [Ideas]
as a user 
i want to be able to sync a folder with a playlist 
in order to not having to add a download myself pointing to a set directory

or

as a user 
i want to be able to save a download task (playlist in a set folder, as there is already a file by file history)
in order to play it again (on startup / on app start up ?)

**Comment by nlogozzo** [comment]
Hmm this sounds like a very interesting and useful feature. I will start thinking about some ways to implement this. 
Will probably go with the second option. Once you download a playlist to a folder, Parabolic can ask if you'd like to keep it synced/updated and everything you start Parabolic it will check and add downloads on startup as needed
Thanks!

**Comment by nlogozzo** [comment]
#681
